docs/style.css

```diff
@@ -0,0 +1,109 @@
+body {
+  overflow-x: hidden;
+}
+.feature-title {
+  text-align: center;
+}
+.left-align {
+  text-align: left;
+}
+.right-align {
+  text-align: right;
+}
+.bg-color-orange {
+  background: rgb(237, 89, 47) !important;
+}
+.browser-info {
+  text-transform: capitalize;
+}
+.feature-content {
+  padding-left: 6em !important;
+  padding-right: 6em !important;
```

Comment:
You have used a lot of `!important`. Was that really `important`? (pun intended)

Reply:
You can use padding and margin classes provided by Vocabulary. See the Spacing documentation in the Storybook.